extensions/guacamole-auth-jdbc/modules/guacamole-auth-jdbc-base/src/main/resources/translations/en.json

```diff
@@ -19,6 +19,7 @@
 
         "FIELD_HEADER_MAX_CONNECTIONS"          : "Maximum number of connections:",
         "FIELD_HEADER_MAX_CONNECTIONS_PER_USER" : "Maximum number of connections per user:",
+        "FIELD_HEADER_WEIGHT"                   : "Connection Weight for Load Balancing:",
```

Comment:
For the sake of consistency, this should follow the same style as the other headers (sentence case, not title case).